src/app/shared/entry-wizard/entry-wizard.component.html

```diff
@@ -30,6 +30,7 @@
           [name]="repo.repositoryName"
           [checked]="repo.present"
           [disabled]="!repo.canDelete && repo.present"
+          [matTooltip]="!repo.canDelete && repo.present ? 'This workflow is full so it cannot be deleted.' : ''"
```

Comment:
I don't understand the tool tip. What is a full workflow?

Reply:
No idea, just wrote what Andrew put in the issue.

Reply:
The description we give users is near the bottom of https://docs.dockstore.org/en/develop/getting-started/dockstore-workflows.html#create-workflow-stubs-from-github-bitbucket-and-gitlab

Reply:
A full workflow is simply not a stub, though in the future ideally there will be no stubs.